src/Frontend/Pages/GameLandingPage.tsx

```diff
@@ -437,9 +437,9 @@ export function GameLandingPage({ match }: RouteComponentProps<{ contract: strin
           );
   
           terminal.current?.printLink(
-            'https://blockscout.com/poa/xdai/',
+            'https://blockscout.com/poa/xdai/optimism',
             () => {
-              window.open('https://blockscout.com/poa/xdai/');
+              window.open('https://blockscout.com/poa/xdai/optimism');
```

Comment:
```suggestion
              window.open('https://blockscout.com/xdai/optimism/');
```

Let's use the link that works :)